tracing/src/macros.rs

```diff
@@ -1057,10 +1057,10 @@ macro_rules! event_enabled {
 /// Tests whether a span with the specified level and target would be enabled.
 ///
 /// This is similar to [`enabled!`], but queries the current collector specifically for
-/// an event, whereas [`enabled!`] queries for an event _or_ span.
+/// a span, whereas [`enabled!`] queries for an event _or_ span.
 ///
 /// See the documentation for [`enabled!]` for more details on using this macro.
```

Comment:
```suggestion
/// See the documentation for [`enabled!`] for more details on using this macro.
```

I also noticed a broken link.